.github/workflows/test.yml

```
@@ -91,6 +91,13 @@ jobs:
         # Custom app-config.local.yaml
         cp -vr configs/app-config/app-config.local.example.yaml configs/app-config/app-config.local.yaml
 
+    - name: Create dynamic plugins directory
+      if: ${{ matrix.composeConfig.name == "dynamic-plugins-root" }}
+      run: |
+        if [[ "${{ matrix.composeConfig.name }}" == "dynamic-plugins-root" ]]; then
+          mkdir -p dynamic-plugins-root
+        fi
```

```suggestion
      if: ${{ matrix.composeConfig.name == 'dynamic-plugins-root' }}
      run: |
        mkdir -p dynamic-plugins-root
```

Looks like double quotes in the `if: ` statement make the workflow invalid: https://github.com/redhat-developer/rhdh-local/actions/runs/15459886129?pr=48